src/main/java/org/broadinstitute/hellbender/engine/SkeletonVariant.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+/**
+ * SkeletonVariant is a minimal implementation of the Variant interface.
+ */
+public class SkeletonVariant implements Variant {
+
+    private SimpleInterval interval;
+    private Boolean isSNP;
```

Comment:
If it's not a SNP, it doesn't follow that it's an indel, so a single boolean is inappropriate here.